UI/MapWnd.cpp

```diff
@@ -394,12 +417,12 @@ namespace {
             ));
 
             if (title) {  // utilize bold font for title label and value
-                m_labels.at(label).first.get()->SetFont(ClientUI::GetBoldFont());
-                m_labels.at(label).second.get()->SetFont(ClientUI::GetBoldFont());
+                m_labels.at(descr).first->SetFont(ClientUI::GetBoldFont());
+                m_labels.at(descr).second->SetFont(ClientUI::GetBoldFont());
             }
 
-            AttachChild(m_labels.at(label).first.get());
-            AttachChild(m_labels.at(label).second.get());
+            AttachChild(m_labels.at(descr).first.get());
```

Comment:
This might be a bad idea... Are you sure the lifetime of the shared_ptr you're getting the raw pointers from here will exist longer than the member variables storing the children in this AttachChild call? If not, then the shared_ptr might be cleared and the pointed to Wnd destructed before the children pointers are cleared, potentially leading to using or attempting to delete an invalid pointer..